wlauto/instrumentation/misc/__init__.py

```diff
@@ -151,16 +151,18 @@ def slow_stop(self, context):
     def update_result(self, context):
         if self.use_tmpfs:
             on_device_tarball = self.device.path.join(self.device.working_directory, self.tarname)
-            on_host_tarball = self.device.path.join(context.output_directory, self.tarname)
-            self.device.execute('{} tar czf {} -C {} .'.format(self.device.busybox,
-                                                               on_device_tarball,
-                                                               self.tmpfs_mount_point),
+            on_host_tarball = self.device.path.join(context.output_directory, self.tarname + ".gz")
+            self.device.execute('{} tar cf {} -C {} .'.format(self.device.busybox,
+                                                              on_device_tarball,
+                                                              self.tmpfs_mount_point),
                                 as_root=True)
             self.device.execute('chmod 0777 {}'.format(on_device_tarball), as_root=True)
-            self.device.pull_file(on_device_tarball, on_host_tarball)
+            self.device.execute('{} gzip {}'.format(self.device.busybox,
+                                                    on_device_tarball))
+            self.device.pull_file("{}.gz".format(on_device_tarball), on_host_tarball)
```

Comment:
For the sake of consistency, I would either change this to use `+ ".gz"`, or update the other occurrences to use `format`.